dream.opam

```diff
@@ -72,6 +72,7 @@ depends: [
   "uri" {>= "4.2.0"}
   "yojson"  # ...
   "ptime" {>= "0.8.1"} # time and date
+  "mirage-clock"
```

Comment:
Not an issue for this PR, but in the future, could you add dependencies in alphabetical order?